mm2src/coins/eth/web3_transport/websocket_transport.rs

```diff
@@ -206,9 +198,6 @@ impl WebsocketTransport {
                     }
 
                     if let Some(id) = inc_event.get("id") {
-                        // just to ensure we don't have outdated entries
-                        response_notifiers.clear_expired_entries();
-
                         let request_id = id.as_u64().unwrap_or_default() as usize;
 
                         if let Some(notifier) = response_notifiers.remove(&request_id) {
```

Comment:
since `remove` doesn't clear entries now. such a pre-clear is needed for correctness.

Reply:
It seems that it will be better to check expiry on remove as well and leave `clear_expired_entries` as private.

Reply:
Done https://github.com/KomodoPlatform/komodo-defi-framework/pull/2232/commits/abfb252ede09f7aa04926627e751363635f90f48